test/unit/src/common/components/repository-row/t_repository-row.js

```diff
@@ -47,7 +47,7 @@ describe('<RepositoryRowView />', () => {
 
     context('configureIsOpen prop is true', function() {
       it('should render unconfigured dropdown', function() {
-        view.setProps({ configureIsOpen: true });
+        view.setProps({ configureIsOpen: true, snap: { snapcraft_data: null } });
```

Comment:
Related to [this change](https://github.com/canonical-ols/build.snapcraft.io/pull/505/files#r106454677).